teamManager/teamManager.py

```diff
@@ -419,6 +421,18 @@ async def teams(self, ctx, *, franchise_tier_identifier: str):
 
         await ctx.send("No tier, franchise, prefix, or GM with name: {0}".format(franchise_tier_identifier))
 
+    @commands.command()
+    @commands.guild_only()
+    async def getTeam(self, ctx: commands.Context, user: discord.Member) -> None:
+        """Fetches current team of discord user and returns the active roster"""
+        log.debug(f"Fetching team name for {user}")
+        team = await self.get_current_team_name(ctx, user)
+        if team is None:
+            await ctx.send(f"{user.display_name} is not currently on a team.")
+            return
```

Comment:
This can be condensed to one line, but doesn't need to be. Technically it would return the message that's sent, but this is directly from the command, so there's no tie in to other code.

Not a necessary change. Unsure about what best practice would be.